src/material/core/ripple/ripple.spec.ts

```diff
@@ -646,6 +646,26 @@ describe('MatRipple', () => {
       expect(rippleTarget.querySelectorAll('.mat-ripple-element').length).toBe(1);
     });
 
+    it('fades out all ripples when disabled input is set', fakeAsync(() => {
+      controller.ripple.launch(0, 0);
+      controller.ripple.launch(0, 0);
+
+      expect(rippleTarget.querySelectorAll('.mat-ripple-element').length).toBe(2);
+
+      tick(enterDuration);
+
+      expect(rippleTarget.querySelectorAll('.mat-ripple-element').length).toBe(2);
+
+      spyOn(controller.ripple, 'fadeOutAll');
```

Comment:
Should this also have `.and.callThrought`? Currently it won't call the original method implementation.

Reply:
Yeah, I think the test currently would always pass because we just wait for the exit duration. The fade out starts immediately because the `pointer` is not down (i.e. `pointerDown = false`).

Reply:
Got it. I've updated the test to have the `mousedown` action so that it should set `pointerDown = true` and `callThrough()` method.